.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
I would handle it separately

Reply:
Fair enough

Reply:
I remember now why I added this.  If you're building on Windows, and spotlessApply, line endings are converted to CRLF for ALL files.  Definitely not what you want.

I can revert this and just leave it as a local file, but this is definitely a useful change for any other developer on Windows.

Reply:
Indeed. Normally there is a global Git config (`core.autocrlf`) that should take care of that. I do not mind if this file is merged though

https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings